src/main/java/com/ibm/cloudant/kafka/common/utils/JavaCloudantUtil.java

```diff
@@ -123,35 +126,46 @@ public static JSONArray batchWrite(String url, String userName, String password,
 	    return result;
 	}
 
-	public static Database getDBInst(String url, String username, String password) {
-		
-		CloudantClient cantClient = null;
-		Database cantDB = null; 
-		
+	public static CloudantClient getClientInstanceFromDBUrl(String url, String username, String
+			password) {
+		CloudantClient cloudantClient = null;
 		// Create a new CloudantClient instance for account endpoint account.cloudant.com
 		// url: https://account.cloudant.com/dbname
 		// length of "://" is 3
 		if (url != null) {
-			String urlWithoutProtocal = url.substring(url.indexOf("://") +3);
-			
+			String urlWithoutProtocal = url.substring(url.indexOf("://") + 3);
```

Comment:
Can we fix protocal to `urlWithoutProtocol`?

Reply:
Should we consider using `new URI(string)` to get the account info and replace `indedOf() + 3`?

Reply:
Fixed in 6107899e20e886a0351496fb6d40b444823a9aaf